packages/uiweb/src/lib/components/space/SpaceWidget/LiveWidgetContent.tsx

```diff
@@ -157,6 +164,11 @@ export const LiveWidgetContent: React.FC<LiveWidgetContentProps> = ({ isJoined,
               <Text color="white" fontSize={'16px'} fontWeight={'600'}>Join this space</Text>
             </Button>
         }
+        {showMembersModal &&
+          <SpaceMembersSectionModal 
+            onClose={() => setShowMembersModal(false)}
+          />
+        }
```

Comment:
```suggestion
        {showMembersModal
          ? <SpaceMembersSectionModal 
            onClose={() => setShowMembersModal(false)}
          />
          : null
        }
```

lets switch to using ternary operators for components. more on this [here](https://steph-ny.medium.com/logical-and-vs-ternary-operator-cdf06ae5d432)

tl;dr:

> use of AND(&&) operator is most ideal when both values being compared are of type Boolean.

>Ternary operator(?) works similar to an if...else code block and is best used to control what you want as the outcome when the condition isn’t met. Also preferred when comparing non-boolean values.